examples/generated/examplecom/repeated_primitive_message_pb.js

```diff
@@ -345,14 +345,14 @@ proto.examplecom.RepeatedPrimitiveMessage.prototype.clearMyDoubleList = function
 
 /**
  * repeated float my_float = 2;
- * @return {!Array.<number>}
+ * @return {!Array<number>}
  */
 proto.examplecom.RepeatedPrimitiveMessage.prototype.getMyFloatList = function() {
-  return /** @type {!Array.<number>} */ (jspb.Message.getRepeatedFloatingPointField(this, 2));
+  return /** @type {!Array<number>} */ (jspb.Message.getRepeatedFloatingPointField(this, 2));
```

Comment:
Which version of protoc are you using? I ask because ideally this generated code should not have changed as it is unrelated to your bazel changes.

Our CI currently uses protoc v3.2.0.